neuralcompression/models/_hific/_hific_encoder.py

```diff
@@ -0,0 +1,79 @@
+import typing
+
+import torch.nn
+
+from ._channel_norm_2d import _channel_norm_2d
+
+
+class HiFiCEncoder(torch.nn.Module):
```

Comment:
Same as above - I think we can avoid any use of `kwargs` with loops and `torch.nn.Sequential`.